modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1154 @@
+
+The docking behaviour of UI elements is usually only triggered when the page is scrolled. This means that the UI element will stay in the same place relative to the page, no matter how much you scroll. However, with the introduction of the 'split' UI mode, it is now possible to dock the UI element on both page and container scroll. This means that the UI element will stay in the same place relative to the container, regardless of how much you scroll the page or the container itself.
+
+[NOTE]
+* If you're using classic mode and sticky toolbars, the toolbar will dock, but it won't be entirely visible in either the window or the scrollable element. This applies to both viewports.
+* In inline mode, the toolbar will also dock, but it won't be entirely visible in either the window or the scrollable element.
+* ShadowDom is partially supported in this case, where the scrollable container is inside the shadow Root. But in cases when the scrollable container is outside the shadow Root (or an ancestor of it) it is not currently supported within versions of {productname} 6.4 or prior.
+
+
+[[changes]]
+== Changes
+
+{productname} 6.4 also incorporates the following changes:
+
+=== The link plugins context menu items will no longer appear for noneditable links
+// TINY-9491
+
+In previous versions of {productname}, the link plugin would display the context menu items for `contenteditable="false"` links.
+
+As a consequence, links such as `table of contents` would open the context menu when `clicked`. The behaviour for the `table of contents`, is not designed to allow links to be modified or removed as the `table of contents` plugin is considered as a `contenteditable="false"` element.
+
+To fix this bug, {productname} 6.4 introduced a new check that now verify if the link plugin is opened, which will now block the context menu on `table of contents` from opening upon when the `toc` is clicked. 
+
+As a result, the context menu for the `table of contents` now behaves as expected in {productname} 6.4.
+
+=== The formatting of contenteditable="false" elements are no longer cloned to new cells while creating new table rows
+// TINY-9449
+
+In previous versions of {productname}, contenteditable="false" elements such as a *merge tags* could clone the styling to a new table row. In some situations, this cloning would happen during the creation of new table row, where the merge tag prefix would unexpectedly clone the formatting from the previous cell to the new cell.
+
+To correct this issue, {productname} added changes to the *clone logic*, and the cursor position API, which will now return `true`, for any passed `contenteditable=false` elements.
```

Comment:
`and the cursor position API, which will now return `true`, for any passed `contenteditable=false` elements.`, perhaps, this deserves a separate change entry if the integrators should be concerned. If not, this should be removed

Reply:
```suggestion
To correct this issue, {productname} added changes to the *clone logic*, to not copy the format for `contenteditable="false"` elements.
```